[2021-03-03 19:24:45,839] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: send_email.send_email 2021-03-03T19:24:42.104984+00:00 [queued]>
[2021-03-03 19:24:45,848] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: send_email.send_email 2021-03-03T19:24:42.104984+00:00 [queued]>
[2021-03-03 19:24:45,848] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-03 19:24:45,848] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-03 19:24:45,848] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-03 19:24:45,854] {taskinstance.py:1063} INFO - Executing <Task(EmailOperator): send_email> on 2021-03-03T19:24:42.104984+00:00
[2021-03-03 19:24:45,857] {standard_task_runner.py:52} INFO - Started process 50 to run task
[2021-03-03 19:24:45,859] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'send_email', 'send_email', '2021-03-03T19:24:42.104984+00:00', '--job-id', '26', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/send_email.py', '--cfg-path', '/tmp/tmpyccaoc4u', '--error-file', '/tmp/tmpv71vktx9']
[2021-03-03 19:24:45,860] {standard_task_runner.py:77} INFO - Job 26: Subtask send_email
[2021-03-03 19:24:45,883] {logging_mixin.py:104} INFO - Running <TaskInstance: send_email.send_email 2021-03-03T19:24:42.104984+00:00 [running]> on host 4c61e128b408
[2021-03-03 19:24:45,913] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=iotoi
AIRFLOW_CTX_DAG_ID=send_email
AIRFLOW_CTX_TASK_ID=send_email
AIRFLOW_CTX_EXECUTION_DATE=2021-03-03T19:24:42.104984+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-03T19:24:42.104984+00:00
[2021-03-03 19:24:45,965] {logging_mixin.py:104} WARNING - /home/airflow/.local/lib/python3.6/site-packages/airflow/providers/sendgrid/utils/emailer.py:122 PendingDeprecationWarning: Fetching Sendgrid credentials from environment variables will be deprecated in a future release. Please set credentials using a connection instead.
[2021-03-03 19:24:46,616] {emailer.py:147} INFO - Email with subject Server connected from 185.14.233.25 is successfully sent to recipients: [{'to': [{'email': 'lukas.kellerstein@gmail.com'}]}]
[2021-03-03 19:24:46,623] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=send_email, task_id=send_email, execution_date=20210303T192442, start_date=20210303T192445, end_date=20210303T192446
[2021-03-03 19:24:46,640] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-03 19:24:46,673] {local_task_job.py:146} INFO - Task exited with return code 0
